### DISCUSSION TITLE
Item Child Count

### DISCUSSION THREAD

**Discussion by GrooveVandal** [microsoft-search/pnp-modern-search#2715] [Q&A]
Is anyone able to provide guidance on how or if I am able to add itemchildcount to the pnp search results cards layout? I have tried the following handlebars {{itemchildcount}} but getting no output

My output is limited to just document set folders

**Comment by kasperbolarsen** [comment]
Please provide some context. What does your query look like, which selected properties have you selected, which version of the solution are you using, are you using SharePoint or Microsoft Search as a data source

**Reply by GrooveVandal** [reply]
sorry for the delay in responding.  Settings are as follows
Sharepoint Search, query template limits to path and content type (document set)
Properties are default (added LinkOfficeChild as suggest below, couldnt get it to work) + some refinable strings
Standard cards setting, - pointed title on cards at a refinable string ( each card is a document set with a Unique ID and Last name from refinable strings)

Other settings are pretty default - no filter web part, no verticals, input from the search box, search query.

I am attempting to highlight on the cards the number of files in the document set, to avoid a user needing to go deeper if the value is 0

**Comment by wobba** [comment]
See my old post https://www.techmikael.com/2014/04/solution-to-displaying-attachments-for.html for a way to get this. You need to request adding `LinkOfficeChild` to the properties requested and then split the string in the template with handlebars to get the count afaik.

**Reply by GrooveVandal** [reply]
I couldn't get this to work, I added the property, and to be blunt, I struggled to find a solution to display the {itemchildcount} in the handle bar expression window.

**Reply by wobba** [reply]
Add `LinkOfficeChild` to properties to retrieve.

![image](https://user-images.githubusercontent.com/51104/217006035-b64542f6-6864-49ae-9b33-1778da2fbb07.png)

Add this to your item template. `{{occurrences item.LinkOfficeChild "https"}}`. This should output the number of attachments as it counts the number of URL's, and they all start with https.
